assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -294,6 +320,18 @@ const baseSelectors = {
 		return accounts;
 	},
 
+	/**
+	 * Gets account summaries.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Array.<Object>|undefined)} An array of Analytics account summaries; `undefined` if not loaded.
+	 */
+	getAccountSummaries( state ) {
+		return state?.accountSummaries;
+	},
```

Comment:
This was only added so that it can be used in the `modules/analytics` `hasProperties` selector correct?